UT4MasterServer.Common/Helpers/EnumHelpers.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using UT4MasterServer.Common.Enums;
+
+namespace UT4MasterServer.Common.Helpers;
+
+public static class EnumHelpers
+{
+	public static bool HasFlagAny<T>(this T @this, T other) where T : struct, Enum
+	{
+		return (Convert.ToUInt64(@this) & Convert.ToUInt64(other)) != 0;
+	}
+
+	public static string[] EnumToStrings<T>(T @enum) where T : struct, Enum
+	{
+		var result = new List<string>();
+		var allNames = Enum.GetNames<T>();
+		var allValues = Enum.GetValues<T>();
+
+		for (int i = 0; i < allNames.Length; i++)
+		{
+			var val = allValues[i];
+
+			if (@enum.HasFlag(val))
+			{
+				result.Add(allNames[i]);
+			}
+		}
+
+		return result.ToArray();
```

Comment:
Why not just return List<T>?

Reply:
To keep everything in most primitive types.
But since it already is a list it is an unneccessery conversion to plain array operation, i'll make it return a list.